src/App.tsx

```diff
@@ -1,39 +1,63 @@
-import React from 'react';
+import { time } from 'console';
+import { timers } from 'cypress/types/jquery';
```

Comment:
what is that?

Reply:
I have deleted them already , they was appeared accidentally when i created timer and click Enter )